sass/_header.scss

```diff
@@ -3,125 +3,114 @@ Header
-		bottom: 0;
-		left: 0;
-		opacity: 0;
-		position: absolute;
-		right: 0;
-		top: 0;
+    right: 0;
+    top: 0;
+    transition: opacity 200ms;
+    z-index: $z-index-header-backdrop;
+
+    .noscroll & {
+      background-color: transparent !important;
+    }
+  }
+
+  &.bg .backdrop {
+    opacity: 1;
+  }
+
+  a {
+    color: #fff;
+
+    &:hover {
+      border-color: #fff;
+    }
+  }
+
+  .site {
+    &-info {
+      transition: opacity 200ms;
+      padding: 20px 0;
+    }
+
+    &-nav {
+      float: right;
+      margin: 20px 0;
+    }
+
+    &-logo {
+      height: 47px;
+      margin: 15px 15px 15px 0;
+      vertical-align: bottom;
+      width: auto;
+
+      @media screen and (max-width: 768px) {
+        margin-right: 10px;
+      }
+
+      &-link {
         transition: opacity 200ms;
-		z-index: $z-index-header-backdrop;
-		
-		.noscroll & {
-			background-color: transparent !important;
-		}
-	}
-	
-	&.bg .backdrop {
-		opacity: 1;
-	}
-	
-    a {
-        color: #fff;
-        
+
         &:hover {
-            border-color: #fff;
+          border-color: transparent;
         }
+      }
     }
-    
-    .site {
-        &-info {
-			transition: opacity 200ms;
-        }
-		
-		&-nav {
-			float: right;
-			margin: 20px 0;
-		}
-        
-        &-logo {
-            height: 47px;
-            margin: 15px 15px 15px 0;
-            vertical-align: bottom;
-            width: auto;
-
-            @media screen and (max-width: 768px) {
-                margin-right: 10px;
-            }
-            
-            &-link {
-                transition: opacity 200ms;
-                
-                &:hover {
-                    border-color: transparent;
-                }
-                
-                + .site-title:empty + .site-description {
-                    display: inline-block;
-                    margin: 28px 0;
-                }
-            }
-        }
-        
-		&-title {
-			display: inline-block;
-			margin: 20px 0;
-            
-            &:empty + .site-description {
-                border: 0;
-                margin: 0;
-                padding: 0;
-            }
-		}
-        
-        &-description {
-            border: 0 solid rgba(255, 255, 255, .25);
-            border-left-width: 1px;
-            color: #fff;
-            @include rem(font-size, 14px);
-            margin-left: 20px;
-            padding-left: 20px;
-            transition: all .2s ease-in-out;
-			vertical-align: middle;
-			
-			&:empty {
-				display: none;
-			}
-            
-            @media screen and (max-width: $width-tablet) {
-                display: none;
-            }
-        }
+
+    &-title {
+      &:empty + .site-description {
+        border: 0;
+        margin: 0;
+        padding: 0;
+      }
+    }
+
+    &-description {
+      @include rem(font-size, 14px);
+      border: 0 solid rgba(255, 255, 255, .25);
+      border-left-width: 1px;
+      color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.